src/streamlink/plugins/dogan.py

```diff
@@ -14,29 +15,52 @@ class Dogan(Plugin):
     Support for the live streams from Doğan Media Group channels
     """
     url_re = re.compile(r"""
-        https?://(?:www.)?
-        (?:teve2.com.tr/(?:canli-yayin|filmler/.*|programlar/.*)|
-           kanald.com.tr/.*|
-           cnnturk.com/canli-yayin|
-           dreamtv.com.tr/canli-yayin|
-           dreamturk.com.tr/canli)
+        https?://(?:www\.)?
+        (?:cnnturk\.com/(?:canli-yayin|video/.*)|
+           dreamturk\.com\.tr/(?:canli|canli-yayin-izle)|
+           kanald\.com\.tr/.*|
+           teve2\.com\.tr/(?:canli-yayin|filmler/.*|programlar/.*))
     """, re.VERBOSE)
-    playerctrl_re = re.compile(r'''<div[^>]*?ng-controller=(?P<quote>["'])(?:Live)?PlayerCtrl(?P=quote).*?>''', re.DOTALL)
+    playerctrl_re = re.compile(r'''<div\s+id="video-element".*>''', re.DOTALL)
```

Comment:
I think you should add a non-greedy quantifier for the dot-match here, so that it only matches the single HTML div element and not the rest of the document.